.devcontainer/Dockerfile

```diff
@@ -1,8 +1,10 @@
 # [Choice] Python version (use -bookworm or -bullseye variants on local arm64/Apple Silicon): 3, 3.13, 3.12, 3.11, 3.10, 3.9, 3.8, 3-bookworm, 3.13-bookworm, 3.12-bookworm, 3.11-bookworm, 3.10-bookworm, 3.9-bookworm, 3.8-bookworm, 3-bullseye, 3.11-bullseye, 3.10-bullseye, 3.9-bullseye, 3.8-bullseye, 3-buster, 3.11-buster, 3.10-buster, 3.9-buster, 3.8-buster
 ARG VARIANT=3.12-bookworm
-FROM python:${VARIANT}
+FROM ghcr.io/astral-sh/uv:python${VARIANT}
```

Comment:
I advocate for using the raw `python` image and download `uv` as a binary exeuctable, it is easier and we would still use the most basic image.

Reply:
I will make this change in the upcoming commit 🙂

Reply:
@sobolevn would you prefer we installed from [PyPi](https://pypi.org/project/uv/) (like we did for pdm) or would you still advocate for installing it as a binary (from the installation script)?

If we are installing as a binary I am unsure if all the base images will have curl (or wget) to download the installation script.

Reply:
`pip` installation can cause problems in theory. I had several when doing so with `poetry`

Reply:
a746281fbee48b1d57907143446b296439f15a60